src/Sarif.Viewer.VisualStudio.Core/Models/SarifErrorListItem.cs

```diff
@@ -85,7 +85,16 @@ public SarifErrorListItem(Run run, int runIndex, Result result, string logFilePa
             this.ResultId = Interlocked.Increment(ref currentResultId);
             this.ResultGuid = result.Guid;
             this.SarifResult = result;
-            ReportingDescriptor rule = result.GetRule(run);
+            ReportingDescriptor rule = null;
+            try
+            {
+                rule = result.GetRule(run);
+            }
+            catch (Exception e)
```

Comment:
Is Console.WriteLine() relevant - should it go to output pane instead?

Reply:
Unsure, this is the try catch that I have in temporarily just so the extension can run end to end. Once I figure out what to do with the rules/tools/extensions problem that I mentioned in the PR description I'll change this out.